package.json

```diff
@@ -96,7 +96,7 @@
   "devDependencies": {
     "@types/jest": "^24.0.15",
     "@types/js-yaml": "^4.0.9",
-    "@types/node": "^15.6.1",
+    "@types/node": "^17.0.41",
```

Comment:
aligns with tag `ts3.9`, using version instead of tag for more predictable lockfile behavior